internal/service/searchindex/resource_search_index_test.go

```diff
@@ -11,18 +11,19 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 )
 
-func TestAccSearchIndex_basic(t *testing.T) {
+func basicTestCase(tb testing.TB) *resource.TestCase {
```

Comment:
knit: i prefer to have the public methods first and the private ones last in the file

Reply:
Done in cc3d9095494f6eb9dbda1c0a592e2a5c825133da ✅